pandora-client-web/src/ui/components/chat/commands.ts

```diff
@@ -368,4 +369,78 @@ export const COMMANDS: readonly IClientCommand<ICommandExecutionContextClient>[]
 			}),
 	},
 	//#endregion
+	//#region Commands for expression changes
+	{
+		key: ['blush'],
+		/*
+			This manipulates the blushing of a player. The main challenge here was the incredilble flexible way,
+			pandora handles assets. Currently we only have one blushing variant, but it may be that we will have
+			other variants with more or less levels than the current one. So this flexibility had to be dealt with
+			when the item was selected. Hopefully the way how 'blush' is extracted from the player can work
+			as a blueprint for others.
+		*/
+		description: 'Increase or decrease your blushing',
+		longDescription: '',
+		usage: 'blush + | - | min | max',
```

Comment:
Should be just `'+ | - | min | max'` as the "blush" is added automatically.

Reply:
Changed.